openwisp_radius/base/models.py

```diff
@@ -816,11 +816,11 @@ class AbstractRadiusBatch(OrgMixin, TimeStampedEditableModel):
         max_length=20,
         help_text=_('Usernames generated will be of the format [prefix][number]'),
     )
-    pdf = models.FileField(
+    pdf_json = JSONField(
```

Comment:
What if in the future we switch from JSON to a different format?

We could also say that now that we have the source data, we are not restricted to generating only PDF, in the future we could deliver that data to the user in other ways. So maybe we could call this field `user_credentials`.

What do you think?

Reply:
Renamed to `user_credentials`